README.md

```diff
@@ -68,7 +68,57 @@ Please note that the date shown is the date that the change was released to the
 
 ---
 
-### 11 April 2024
+### 16 April 2024
+
+The following changes are now available in Production.
+
+#### individual-calculations-api v5
+
+`Retrieve a Self Assessment Tax Calculation` includes the following properties:
+- `calculation.reliefs.giftAidTaxReductionWhereBasicRateDiffers`
+- `calculation.taxCalculation.incomeTax.giftAidTaxChargeWhereBasicRateDiffer` and
+- `taxDeductedAtSource.taxTakenOffTradingIncome`.
+
+In addition, `cashOrAccruals` and `documentDetails.documentDueDate` are now mandatory.
+
+#### business-details-api v1
+
+* `Retrieve Business Details` includes the object `quarterlyTypeElection` and 
+* a new endpoint, `Create and Amend Quarterly Period Type for a Business` is now available.
+
+#### self-assessment-accounts-api
```

Comment:
Is there a reason why this does not have the version number at the end whilst the rest of the APIs do?